cli/pkg/digger/digger.go

```
@@ -533,19 +534,19 @@ func reportPlanSummary(reporter reporting.Reporter, summary string) {
 }
 
-func reportEmptyPlanOutput(reporter reporting.Reporter, projectId string) {
+func reportEmptyPlanOutput(projectName string, reporter reporting.Reporter, projectId string) {
 	identityFormatter := func(comment string) string {
 		return comment
 	}
-	_, _, err := reporter.Report("→ No changes in terraform output for "+projectId, identityFormatter)
+	err := reporter.Report(projectName, "→ No changes in terraform output for "+projectId, identityFormatter)
 	// suppress the comment (if reporter is suppressible)
-	reporter.Suppress()
+	reporter.Suppress("")
```

golangci-lint (1.62.2)

Error return value of `reporter.Suppress` is not checked

(errcheck)